.github/workflows/linux.yml

```diff
@@ -1,5 +1,10 @@
 name: Check Linux Packages
 on:
+  push:
```

Comment:
This feels like a waste of GitHub actions minutes, why would we run GitHub actions on a change that has already been merged?

Reply:
The main purpose was to build it on the other branches (feature + bugfix) on a direct push (e.g., on forked repos). If changes to master are only performed by PRs the master branch could be dropped for the repo in the adoptium organisation.

But perhaps people working on forks do not create feature- or bugfix-branches but work directly on the master branch on their fork. They probably also want to build (on a push)?

Reply:
I don't see the need for this personally, as soon as you raise the pull request to the master branch the GitHub actions checks will start running. I've personally never used a `feature/xx` schema and if developers really wanted to they could maintain this patch in their own fork.

Reply:
Well, we discussed this in the community call and most people found this useful. I think it was finally checked by @smlambert with the PMC?

Reply:
related issue created for discussion: https://github.com/adoptium/adoptium/issues/203 (based on bringing the topic to the PMC call

Reply:
And the respective commit referenced adoptium/adoptium#203

Reply:
I've clearly missed this conversation so apologies for not providing any feedback at an earlier date. To me, it's not clear what value this provides to contributors. Why would they need to run GitHub actions on their own fork when it can be run directly on the pull request so that reviewers can see the results? If you really see value in this then my proposition would be to prevent it from running on pushes to the Adoptium repo (e.g only run on forks). That said this wasn't discussed with the PMC so I'd like to get their thoughts before we implement such a feature.

Reply:
As a contributor I would like to perform all tests on my own GH account before raising a PR. Isn't this motivation enough? Of course I could add the respective commit to each of my branches, test it on my GH account and remove it before raising the PR. But this is some extra effort which I would like to spare myself and also other contributors who would like to perform testing on GH and are not that familiar with Git and GH actions. I even volunteered to write some developer documentation how to do this.

The change makes it easy. In the end, I think it doesn't do any harm to anyone, as no one needs to make use of it. But it helps at least me (and perhaps others). If you are concerned about some extra builds on the master branch, this could be resolved, e.g., by an additional guard condition to check whether the GH organisation is not `adoptium`. 

In the the community session three weeks ago, no one objected, some appreciated the idea. The only issue was that perhaps other repos have some branching policy in place already (cf. adoptium/adoptium#203). We investigated all repos, there is no similar pattern in place.

So why not do it this way, or do you have a better idea? I love to learn and hear constructive counter-proposals.

Reply:
Thanks for the additional context (it really helps when making these decisions). Here is what I'd propose:

1. Remove the `branches: [ master, 'feature/**', 'bugfix/**' ]` block from the `push` trigger (I've seen very few people use this name schema and it make no sense to restrict people by branch name.
2. Add the following code block to the `check-ca-certificates`, `check-packages` and `validation` jobs:
```yaml
if: github.event_name == 'pull_request' || github.repository_owner != 'adoptium'
```
This way it will only trigger the job on people's forks or on the Adoptium pull requests (which seems like the best approach here and saves burning through Adoptium executor minutes.